### ISSUE TITLE
Issues using Laravel 5.2

### ISSUE THREAD

**Issue by clin407** [GeneaLabs/laravel-mixpanel#27]
Followed the instructions to install this package using 

composer require genealabs/laravel-mixpanel:~0.5.1

When I added 

```        
'GeneaLabs\LaravelMixpanel\Providers\LaravelMixpanelServiceProvider',
```
to config/app.php and load a page, it says 
```
ReflectionException in Container.php line 734:
Class does not exist
```

Any ideas why?

**Comment by mikebronner** [comment]
Can you provide the full stack trace? That would help me identify what the root cause is. :)

**Comment by clin407** [comment]
@mikebronner Laravel 5.2.45

```
in Container.php line 734
at ReflectionClass->__construct('') in Container.php line 734
at Container->build(null, array()) in Container.php line 629
at Container->make(null, array()) in Application.php line 697
at Application->make(null) in LaravelMixpanelServiceProvider.php line 20
at LaravelMixpanelServiceProvider->boot(object(Request), object(SessionGuard), object(LaravelMixpanel))
at call_user_func_array(array(object(LaravelMixpanelServiceProvider), 'boot'), array(object(Request), object(SessionGuard), object(LaravelMixpanel))) in Container.php line 507
at Container->call(array(object(LaravelMixpanelServiceProvider), 'boot')) in Application.php line 757
at Application->bootProvider(object(LaravelMixpanelServiceProvider)) in Application.php line 740
at Application->Illuminate\Foundation\{closure}(object(LaravelMixpanelServiceProvider), '37')
at array_walk(array(object(EventServiceProvider), object(RoutingServiceProvider), object(AuthServiceProvider), object(CookieServiceProvider), object(DatabaseServiceProvider), object(EncryptionServiceProvider), object(FilesystemServiceProvider), object(FoundationServiceProvider), object(PaginationServiceProvider), object(SessionServiceProvider), object(ViewServiceProvider), object(AccessServiceProvider), object(ServiceProvider), object(GeneratorsServiceProvider), object(AppServiceProvider), object(AuthServiceProvider), object(EventServiceProvider), object(HistoryServiceProvider), object(RouteServiceProvider), object(UtilitiesServiceProvider), object(LogViewerServiceProvider), object(NoCaptchaServiceProvider), object(GravatarServiceProvider), object(ServiceProvider), object(JavaScriptServiceProvider), object(ActiveServiceProvider), object(BackupServiceProvider), object(HtmlServiceProvider), object(ExcelServiceProvider), object(DatatablesServiceProvider), object(ServiceProvider), object(ServiceProvider), object(ImageServiceProvider), object(GeoIPServiceProvider), object(LaradropServiceProvider), object(TaggingServiceProvider), object(MediaLibraryServiceProvider), object(LaravelMixpanelServiceProvider), object(MailServiceProvider), object(QueueServiceProvider), object(TranslationServiceProvider), object(RouteServiceProvider), object(ValidationServiceProvider), object(MacroServiceProvider), object(HashServiceProvider)), object(Closure)) in Application.php line 741
at Application->boot() in BootProviders.php line 17
at BootProviders->bootstrap(object(Application)) in Application.php line 203
at Application->bootstrapWith(array('Illuminate\Foundation\Bootstrap\DetectEnvironment', 'Illuminate\Foundation\Bootstrap\LoadConfiguration', 'Illuminate\Foundation\Bootstrap\ConfigureLogging', 'Illuminate\Foundation\Bootstrap\HandleExceptions', 'Illuminate\Foundation\Bootstrap\RegisterFacades', 'Illuminate\Foundation\Bootstrap\RegisterProviders', 'Illuminate\Foundation\Bootstrap\BootProviders')) in Kernel.php line 232
at Kernel->bootstrap() in Kernel.php line 127
at Kernel->sendRequestThroughRouter(object(Request)) in Kernel.php line 99
at Kernel->handle(object(Request)) in index.php line 53
```

**Comment by clin407** [comment]
Any ideas on this?

**Comment by mikebronner** [comment]
Try installing version 0.5.3 of the package, it includes numerous fixes / changes from 0.5.1. Let me know how that goes.

**Comment by clin407** [comment]
Doesn't work. Oh well =\
